docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md

```diff
@@ -0,0 +1,81 @@
+## CTCGreedyDecoderSeqLen <a name="CTCGreedyDecoderSeqLen"></a> {#openvino_docs_ops_sequence_CTCGreedyDecoderSeqLen_6}
+
+**Versioned name**: *CTCGreedyDecoderSeqLen-6*
+
+**Category**: Sequence processing
+
+**Short description**: *CTCGreedyDecoderSeqLen* performs greedy decoding of the logits provided as the first input. The sequence lengths are provided as the second input.
+
+**Detailed description**:
+
+This operation is similar to TensorFlow CTCGreedyDecoder [TensorFlow CTCGreedyDecoder](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_greedy_decoder).
```

Comment:
Change "This operation is similar to TensorFlow CTCGreedyDecoder [TensorFlow CTCGreedyDecoder](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_greedy_decoder)." to "This operation is similar to the [TensorFlow CTCGreedyDecoder](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_greedy_decoder)."